marathon/tests/test_unit.py

```diff
@@ -74,3 +80,38 @@ def test_get_instance_config(check):
     _, acs_url, _, tags, label_tags = check.get_instance_config(instance)
     assert tags == ['foo:bar']
     assert label_tags == ['label_foo']
+
+
+@pytest.mark.parametrize(
+    'test_case, init_config, extra_config, expected_http_kwargs',
+    [
+        (
+            "new config",
+            {},
+            {'timeout': 5, 'username': 'foo', 'password': 'bar', 'tls_verify': False},
+            {'timeout': 5, 'auth': ('foo', 'bar'), 'verify': False},
+        ),
+        (
+            "legacy config",
+            {'default_timeout': 3},
+            {'user': 'foo', 'password': 'bar', 'disable_ssl_validation': True},
+            {'timeout': 3, 'auth': ('foo', 'bar'), 'verify': False},
```

Comment:
Could you also add a test for `read_timeout` and `connect_timeout` ?

https://github.com/DataDog/integrations-core/blob/70afceeee691fbdcfa38c9806413da07d0ab6e9b/marathon/datadog_checks/marathon/marathon.py#L53